gpt4all-chat/CMakeLists.txt

```diff
@@ -230,9 +234,11 @@ qt_add_qml_module(chat
       icons/edit.svg
       icons/eject.svg
       icons/email.svg
+      icons/file-doc.svg
       icons/file-md.svg
       icons/file-pdf.svg
       icons/file-txt.svg
+      icons/file-xls.svg
```

Comment:
The file-doc, file-xls, and paperclip icons have not been committed, so I can't build this.